mocks/mock_scratch_bucket_creator.go

```diff
@@ -1,19 +1,5 @@
-//  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Add the copyright back (it gets deleted by mockgen when you generate mock again)